lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -1378,13 +1379,31 @@ class LineChartPainter extends AxisChartPainter<LineChartData> {
     /// draw the texts one by one in below of each other
     var topPosSeek = tooltipData.tooltipPadding.top;
     for (var tp in drawingTextPainters) {
-      final drawOffset = Offset(
-        rect.center.dx - (tp.width / 2),
-        rect.topCenter.dy +
-            topPosSeek -
-            textRotationOffset.dy +
-            rectRotationOffset.dy,
-      );
+      double yOffset = rect.topCenter.dy +
+          topPosSeek -
+          textRotationOffset.dy +
+          rectRotationOffset.dy;
+
+      final ui.Offset drawOffset;
+
+      if (tp.textAlign.getFinalHorizontalAlignment(tp.textDirection) ==
```

Comment:
We can replace these `if`, `if else`, `else` with a switch case. (I think it's more readable)

Reply:
Somthing like this:
```dart
double xOffset;
switch (tp.textAlign.getFinalHorizontalAlignment(tp.textDirection)) {
  case HorizontalAlignment.left:
    xOffset = rect.left + tooltipData.tooltipPadding.left;
    break;
  case HorizontalAlignment.right:
    xOffset = rect.right - tooltipData.tooltipPadding.right - tp.width;
    break;
  default:
    xOffset = rect.center.dx - (tp.width / 2);
    break;
}
drawOffset = Offset(xOffset, yOffset);
```

Reply:
ok.